packages/adblocker-webextension/adblocker.ts

```diff
@@ -396,16 +395,17 @@ export class WebExtensionBlocker extends FiltersEngine {
 
       // Inject scripts from content script
       if (scripts.length !== 0) {
-        sendResponse({
+        return {
           active,
           extended,
           scripts,
           styles: '',
-        });
+        };
       }
     }
 
     await Promise.all(promises);
+    return undefined;
```

Comment:
Nit. Is that required?
```suggestion
```

Reply:
The explicit return is enforced by the Linter; otherwise it will complain about missing returns on some paths.